collect_app/src/main/res/values/strings.xml

```diff
@@ -511,4 +511,5 @@
     <string name="server_settings_title">Server</string>
     <string name="type">Type</string>
     <string name="form_entry">Uncheck to hide from Form Entry</string>
+    <string name="open">Open in external app</string>
```

Comment:
Calling this `open_external_video` or something like that will help with readability.